pillar.example

```diff
@@ -32,6 +32,10 @@ postgres:
     hard: 64000
 
   # POSTGRES
+  # Non-standard port to use for the cluster
+  # Only set if port `5432` is not appropriate
+  port: 5433
+
```

Comment:
With what I said above, we could provide top level `postgres.port` Pillar setting. This would be reused directly in the server SLS and automatically enable correct port resolution for all Salt routines related to PostgreSQL.

But just better to keep it default mentioning it could be changed.